spec.html

```diff
@@ -49152,23 +49125,23 @@ <h1>Valid Executions</h1>
 
   <emu-clause id="sec-races">
     <h1>Races</h1>
-    <p>For an execution _execution_, two events _E_ and _D_ in SharedDataBlockEventSet(_execution_) are in a race if the following algorithm returns *true*.</p>
+    <p>For an execution _execution_, two events _E_ and _D_ in SharedDataBlockEventSet(_execution_) are in a <em>race</em> if the following algorithm returns *true*.</p>
     <emu-alg>
       1. If _E_ and _D_ are not the same Shared Data Block event, then
-        1. If the pairs (_E_, _D_) and (_D_, _E_) are not in _execution_.[[HappensBefore]], then
+        1. If it is not the case that _E_ happens-before _D_ in _execution_ and _D_ happens-before _E_ in _execution_, then
```

Comment:
The scope of the negation is ambiguous.
```
it is not the case that A and B
```
could mean
```
(it is not the case that A) and B
```
or
```
it is not the case that (A and B)
```
(Though I'm guessing it's fairly clear to anyone interested in this section that the latter is intended.)

Reply:
I'll add a "both".

Reply:
I wouldn't be against adding an "Unless" form for cases like this.